### DISCUSSION TITLE
Import-Excel Affected by Powershell 7.4 Constrained Language Mode

### DISCUSSION THREAD

**Discussion by gtatelive** [dfinke/ImportExcel#1549] [Q&A]
Wanted to share an issue that was encountered today after updating to PowerShell 7.4. 

ImportExcel module wasn't working as expected after the upgrade. The cmdlett ran, but nothing was returned.

The shell was being ran in constrained language mode without any restrictions placed. The expected function was for ImportExcel module to continue functioning yet it didn't. There weren't any overt errors thrown however after enabling and reviewing "PowershellCore/Analytic" log we could see the errors saying the module or script was untrusted and wouldn't be allowed to run.

Further investigation found the following issue on the PowerShell Git which links to a possible fix in 7.4.1 
[Constrained Language Audit mode breaks az.resources #20785](https://github.com/PowerShell/PowerShell/issues/20785)

**Comment by dfinke** [comment]
Thanks @gtatelive of using `ImportExcel` and tracking that down. I don't use it in constrained language mode.

Sounds like we're waiting on the PowerShell team
